dev/tasks/python-wheels/github.osx.yml

```
@@ -60,7 +60,7 @@ jobs:
 
       - name: Retrieve VCPKG version from arrow/.env
         run: |
-          vcpkg_version=$(cat "arrow/.env" | grep "VCPKG" | cut -d "=" -f2 | tr -d '"')
+          vcpkg_version=$(. "arrow/.env" | echo "${VCPKG}")
```

That doesn't seem right?
```console
$ bash -c '. ".env" | echo "${VCPKG}"'

$ bash -c '. ".env" && echo "${VCPKG}"'
943c5ef1c8f6b5e6ced092b242c8299caae2ff01
```


```suggestion
          vcpkg_version=$(. "arrow/.env" && echo "${VCPKG}")
```

---

Oh... Thanks! I don't know why I used `|` here...